assets/product-info.js

```diff
@@ -364,6 +364,7 @@ if (!customElements.get('product-info')) {
       }
 
       updateQuantityRules(sectionId, html) {
+        if (!this.quantityInput) return;
```

Comment:
Connected it to the store and it looks good 👍 https://admin.shopify.com/store/os2-demo/themes/163387473942/editor

However I think we should do this check in even before running the `updateQuantityRules` function 🤔  

What do you think? cc @ludoboludo

Reply:
I think it's fine as is 🤔 We do the same approach within the `initQuantityHandlers()` method 👍 return.

And in a few other places in the theme. There is definitely a mix of approaches but I think it's good here since the method is called a couple times. Instead of adding this check twice having it part of the method seem good because it takes care of all the logic itself.